tests/backstop/engine_scripts/puppet/onReady.js

```diff
@@ -1 +1,115 @@
-module.exports = async () => {};
+/**
+ * Custom onReady script for BackstopJS.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const POLLING_INTERVAL_MS = 100;
+const FONT_WAIT_TIMEOUT_MS = 5000;
+
+/**
+ * Creates a promise that resolves after a specified delay.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} ms The delay in milliseconds.
+ * @return {Promise<void>} A promise that resolves after the delay.
+ */
+const delay = ( ms ) => new Promise( ( resolve ) => setTimeout( resolve, ms ) );
```

Comment:
Let's call this `waitForTimeout()`, or something similar, to better align with naming conventions elsewhere in the codebase.

Furthermore, please rename `ms` to `milliSeconds` to adhere to our convention of generally avoiding abbreviations in identifiers.